docs/ctapipe_api/coordinates/index.rst

```diff
@@ -35,6 +36,13 @@ The following special frames are defined for CTA:
 they can be transformed to and from any other `astropy.coordinates` frame, like
 `astropy.coordinates.AltAz` or `astropy.coordinates.ICRS` (RA/Dec)
 
+The two different coordinate frames are shown here:
+
+.. plot:: ../examples/plot_camera_frames.py
+    :include-source:
+
+The `CameraFrame` is used internally and is used by H.E.S.S. and `sim_telarray`,
+the `EngineeringCameraFrame` is used by `MAGIC` and `FACT`.
```

Comment:
Actually, in HESS we also use the EngineeringFrame.  The HESS analysis framework (SASH) is independent of simtelarray and it's built-in analysis (read_hess).  I think the SMASH simulations also use the EngineeringFrame, but I'd have to check to be sure.  Might be better to say the CameraFrame comes from the sim_telarray conventions, which in turn come from the conventions in Corsika.

Reply:
Okay, then I think de should also discuss if es really want to call it `CameraFrame` and `EngineeringCameraFrame` and which One de really want to use as internal discription.

Reply:
I think as long as we document what each means, it's fine to keep it as is.  Engineering makes the most sense for a frame where the telescope is parked and where somebody might be working on the camera, to replace a module or something.